### ISSUE TITLE
Building with VS 2015

### ISSUE THREAD

**Issue by walbourn** [openexr/openexr#205]
I'm trying to build OpenEXR with Visual C++ and having no luck. I took the cmake route per the instructions [here](https://github.com/openexr/openexr/blob/develop/OpenEXR/README.cmake.txt), but they don't work. These get me close, but I still cannot get it to build:

```
cd /d d:\
git clone https://github.com/openexr/openexr.git
cd openexr\IlmBase
md vsbuild
cd vsbuild
cmake -G"Visual Studio 14 Win64" ..
```

Build ilmbase.sln (works, but generates a ton of warnings)

```
cd ..\..\openexr
md vsbuild
cd vsbuild
cmake -DZLIB_ROOT=D:\zlib-1.2.8 -DZLIB_LIBRARY=D:\zlib-1.2.8\contrib\vstudio\vc11\x64\ZlibDllRelease\zlibwapi.lib -DILMBASE_PACKAGE_PREFIX=D:\openexr\ilmbase\vsbuild -G "Visual Studio 14 Win64" ..
```

Build openexr.sln, and I get numerous errors like:

> D:\openexr\OpenEXR\IlmImf\b44ExpLogTable.cpp(52): fatal error C1083: Cannot open include file: 'half.h': No such file or directory
> 
>  d:\openexr\openexr\ilmimf\ImfInt64.h(44): fatal error C1083: Cannot open include file: 'ImathInt64.h': No such file or directory

**Comment by walbourn** [comment]
BTW, it would be super-helpful if you guys just published a [NuGet](https://www.nuget.org/) package for Windows developers

**Comment by caron** [comment]
@walbourn You might be interested in my appveyor support for OpenVDB (it includes building OpenEXR from source). I was actually interested in forking OpenEXR and adding support for appveyor also and maybe we can get NuGet package published too?

https://github.com/caron/openvdb/tree/v3.2.0-win-appveyor

particularly this part...
https://github.com/caron/openvdb/blob/v3.2.0-win-appveyor/appveyor.yml#L57

**Comment by meshula** [comment]
The vfxpro99 OpenEXR script for USD works a treat.

https://github.com/vfxpro99/usd-build-club/blob/master/prerequisites-vc140-x64/openexr.cmd

**Comment by walbourn** [comment]
Thank you so much. That script rocks!

**Comment by mancoast** [comment]
@caron 
The nuget request is complete: https://github.com/openexr/openexr/pull/218

For package installation:
Install-Package openexr-msvc14-x86
Install-Package openexr-msvc14-x64

thanks,
coast

**Comment by shreyaj** [comment]
I am having the same issue. I configured IlmBase and OpenEXE successfully. Now, I try to build openexe but it fails showing that it cannot find any of the include files. I added the include files path (C:\Program\include) and lib files path (C:\Program\lib) to my user environment variables and to each project's properties in Visual studio but nothing works. I keep getting the error that half.h cannot be found when I do "cmake --build . --target install --config Release".
Did you guys end up building OpenEXE?
PLEASE HELP!

**Comment by meshula** [comment]
I believe this was fully resolved in #218.
